### PR TITLE
Android Login Feature

### LINKED ISSUE #385: Create Login Screen Interface
Create a login screen that works as first screen in startup of the app.

### PR DESCRIPTION
**Pull request by amiapmorais**
fix #385 #389 

Created the first Android app screen.

- [ ] **Model testing.**
- [ ] **Request testing.**
- [ ] **System testing.**
- [x] **No tests required.**

### CODE REVIEW

File: android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package com.example.espoo.view
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Toast
+import com.example.espoo.R
```
Comment (edimossilva) [inline android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt:9]: com.example?

**Review by edimossilva — APPROVED** [review]
Just change the package and it is cool

**Comment by edimossilva** [comment]
Can you see code climate complains?

**Comment by amiapmorais** [comment]
This is my "Hello World" in Kotlin language. I have a to learn about. If anyone has any question, please ask. I will be happy to answer.

File: android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt (hunk 1 above)
Comment (amiapmorais) [reply android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt:9]: I didn't see this. I will refactor to just com.espoo, or do you think we can use another package name?

Comment (edimossilva) [reply android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt:9]: Agree

**Comment by amiapmorais** [comment]
@edimossilva could you explain about what is the code climate complains?

**Comment by amiapmorais** [comment]
I just saw the code climate. It's new for me. I'll fix that.

**Comment by edimossilva** [comment]
> I just saw the code climate. It's new for me. I'll fix that.

Sometimes code climate complains about things that are not relevant and you can configure it to ignore them

**Pull request merged**